**Command**
```
./gradlew clean :core:compileScala -PkeepAliveMode=session
```

**Before**
```
> Task :core:compileScala
Unexpected javac output: warning: [path] bad path element "/Users/hongyi/IdeaProjects/kafka/core/build/classes/java/main": no such file or directory
error: warnings found and -Werror specified
1 error
1 warning.
[Warn] : javac: [path] bad path element "/Users/hongyi/IdeaProjects/kafka/core/build/classes/java/main": no such file or directory
javac exited with exit code 1

> Task :core:compileScala FAILED

BUILD FAILED in 32s
88 actionable tasks: 88 executed
```

**After**

```
> Configure project :
Starting build with version 4.2.0-SNAPSHOT (commit id 2fb9e503) using Gradle 9.1.0, Java 23 and Scala 2.13.17
Build properties: ignoreFailures=false, maxParallelForks=12, maxScalacThreads=8, maxTestRetries=0

BUILD SUCCESSFUL in 37s
88 actionable tasks: 88 executed
```

--- a/build.gradle
+++ b/build.gradle
@@ -138,8 +138,7 @@ ext {
       options.compilerArgs << "-Werror"
       if (userKeepAliveMode == KeepAliveMode.SESSION){
         // Suppress warning triggered under keepAliveMode=SESSION during joint compilation
-        //   Unexpected javac output: warning: [path] bad path element ".../core/build/classes/java/main": no such file or directory
-        // This becomes an error under -Werror
+        // Unexpected javac output: warning: [path] bad path element ".../core/build/classes/java/main": no such file or directory
         options.compilerArgs << "-Xlint:-path"
       }
     }

